checker/src/main/java/org/checkerframework/checker/builder/qual/CalledMethods.java

```diff
@@ -0,0 +1,25 @@
+package org.checkerframework.checker.builder.qual;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * For compatibility with Lombok, retain the old annotations it generates.
```

Comment:
Maybe rephrase?  Something like "An old version of the annotation, retained for compatibility with Lombok"

Can we add a `@Deprecated` annotation here?

Reply:
We decided against it when we merged https://github.com/kelloggm/object-construction-checker/pull/122. The reason is that it would cause deprecation warnings in delomboked code, which is annoying. I'll add a comment to that effect.

Reply:
Ok.  I would still rephrase the first sentence of this Javadoc comment